assets/js/commento.js

```diff
@@ -215,10 +226,7 @@
 
 
     var _getComments = function() {
-        var data = {
-            "url": document.location
-        };
-        post(_api.get, data, function(reply) {
+        get(_api.get + '?url=' + document.location, function(reply) {
```

Comment:
Or is that taken care of by `XMLHttpRequest`? (although I'm skeptical because it can't possibly know which part is to be encoded)